ap/config/categories.py

```diff
@@ -11,6 +11,13 @@ def add_categories(config: od.Config) -> None:
     """
     Adds all categories to a *config*.
     """
+    config.add_category(
+        name="incl",
+        expression="sel_incl",
+        id=99,
+        label="inclusive",
+        selection="sel_incl",
+    )
```

Comment:
Btw: I made a mistake in order. Categories should not have expressions but only selections. I reverted this and adapted the code in a commit to this branch. Hope that is ok for you.